tests/jballerina-unit-test/src/test/resources/test-src/javainterop/basic/instance_method_tests.bal

```diff
@@ -1,4 +1,21 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
2019?

Reply:
The file was created in 2019, added the license header when I modified it